Kitodo/src/main/webapp/WEB-INF/resources/css/kitodo.css

```diff
@@ -3023,6 +3029,22 @@ Column content
     position: relative;
 }
 
+.thumbnail-parent[data-mime-type^="video"] {
+    width: 185px;
+    height: auto;
+}
+
+
+#imagePreviewForm\:thumbnailStripe .thumbnail-parent[data-mime-type^="video"] {
```

Comment:
It is a common way to use such selector to style by attribute values. If there is a convention or this is not wanted by the majority then I take it out. The advantage here is that no `IF ELSE` has to be used in the template to render the class.

Reply:
As far as I know we don't have any conventions regarding this. I think it's fine to use data attributes in CSS selectors, especially when a CSS class would be a redundant information.

Reply:
According to Codacy the regex ( `data-mime-type^="video"`) in the CSS selector can negatively impact performance and should be avoided (see Codacy report of current build for details). 

Since this code is part of the metadata editor, we should probably be extra vigilant and strive not to loose the performance improvements we gained in the last year. So perhaps a CSS class would be preferable after all?